lib/log/json_formatter.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module Log
+  class JSONFormatter
+
+    def call(severity, time, progname, msg)
+      log_item = {
```

Comment:
I didn't, because it's added by logspout anyways. I agree though, that it would probably make more sense to add the timestamp here and not there.